added sub-api and minimark (#98)
substrate-developer-hub/awesome-substrate: README.md

- [Python Interface](https://github.com/polkascan/py-substrate-interface) - Maintained by [Polkascan Foundation](https://polkascan.org/).
- [Rust API Client](https://github.com/scs/substrate-api-client) - Rust client maintained by [Supercomputing Systems AG](https://www.scs.ch/).
- [Subscan Go Utilities](https://github.com/itering/subscan-essentials) - SS58 and more, developed by Subscan.

> - [Sub-Api](https://github.com/kodadot/packages) - Logical and friendly wrapper for Substrate API maintained & used by KodaDot in production.

Reviewer: ```suggestion
- [Sub-Api](https://github.com/kodadot/packages/tree/main/sub-api) - Friendly wrapper for Polkadot.js API maintained by KodaDot
```

- [Jupiter](https://github.com/patractlabs/jupiter) - Testnet for smart contracts written for the FRAME Contracts pallet and ink!.
- [Megaclite](https://github.com/patractlabs/megaclite) - Zero-knowledge tools for the Polkadot ecosystem.
- [Metadata Portal](https://nova-wallet.github.io/metadata-portal/) - A self-hosted webpage that shows the latest metadata and chain specs for any given network.

> - [Minimark](https://github.com/kodadot/packages) - Implementation of RMRK v1/v2 protocol used by KodaDot Indexers in production

Reviewer: ```suggestion
- [Minimark](https://github.com/kodadot/packages) - Implementation of RMRK NFT v1/v2 protocol maintained by KodaDot
```
